app/javascript/components/mentoring/inbox/TrackList.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import { GraphicalIcon, TrackIcon } from '../../common'
+import { useDropdown } from '../../dropdowns/useDropdown'
+
+type Track = {
+  slug: string
+  title: string
+  iconUrl: string
+  count: number
+}
+
+const TrackFilter = ({
+  title,
+  iconUrl,
+  count,
+  onChange,
+  checked,
+}: Track & { onChange: () => void; checked: boolean }) => {
+  return (
+    <React.Fragment>
+      <label className="c-checkbox-wrapper">
+        <input type="checkbox" onChange={onChange} checked={checked} />
+        <div className="c-checkbox">
+          <GraphicalIcon icon="checkmark" />
+        </div>
+        <TrackIcon iconUrl={iconUrl} title={title} />
+        <div className="title">{title}</div>
+        <div className="count">{count}</div>
+      </label>
+    </React.Fragment>
+  )
+}
+
+export const TrackList = ({
+  tracks,
+  value,
+  setTrack,
+}: {
+  tracks: Track[]
+  value: string | null
+  setTrack: (value: string | null) => void
+}): JSX.Element => {
+  const {
+    buttonAttributes,
+    panelAttributes,
+    listAttributes,
+    itemAttributes,
+  } = useDropdown(tracks.length)
+  const selected = tracks.find((track) => track.slug === value) || tracks[0]
```

Comment:
Are these the tracks that a mentor is mentoring? I guess there will always be at least one?

Reply:
If there isn't then the attribute shouldn't appear at all. I'll discuss this with Tai.